README.md

```diff
@@ -72,31 +72,30 @@ Using Arc is not necessary to deploy a DAO (you can do this with Arc.js currentl
 
-**Arc.js is a JavaScript library that facilitates access to Arc contracts without having to directly interact with the Ethereum blockchain.** Much like the Web3.js library serves as the connection for JavaScript/TypeScript developers to the Ethereum network, the Arc.js library connects Arc with scripts, applications, DAO interfaces, or any other program that knows how to work with JavaScript/TypeScript coming from an NPM module in ES6 CommonJs module format.
+**Client is a library that facilitates access to Arc contracts without having to directly interact with the Ethereum blockchain.** It provides functions to interact with DAOstack contracts to vote, propose, stake and execute proposals.
 
-Using Arc.js, JavaScript/TypeScript developers can easily write scripts or applications which can deploy DAOs, interact with existing DAOs, submit proposals to DAOs, vote and stake on proposals, execute the resulting decisions, manage agent reputations, and basically do anything a DAO is capable of doing.
+Client library is also a wrapper around [DAOstack subgraph](https://github.com/daostack/sugbraph). It enable developers to interact with DAOstack subgraph and execute various generic graph queries to access proposals, daos and other complex entities
 
-Should you chose to leverage Arc.js in your application, you can find helpful documentation in the [complete Arc.js documentation](https://daostack.github.io/arc.js/).
+Using Client, JavaScript/TypeScript developers can easily write scripts or applications which can interact with existing DAOs, submit proposals to DAOs, vote and stake on proposals, execute the resulting decisions, manage agent reputations
 
-Should you choose to develop inside Arc.js itself, you can find helpful information in the [documentation for Arc.js developers](https://github.com/daostack/arc.js/blob/devDocs/docs/DeveloperDocs.md).
-
-You can find NodeJs scripts that use Arc.js to perform a variety of functions in the [Arc.js-scripts Github repository](https://github.com/daostack/arc.js-scripts).
+## [Subgraph](https://github.com/daostack/subgraph)
+**Subgraph indexes the blockchain data and stores it in postgres database for easy and quich access.** The data store can be queried by GraphQL endpoints. DAOstack subgraph is based on the graphprotocol checkout [TheGraph](https://thegraph.com/docs/quick-start) for more details.
 
 ### Should I work at this level?
 
-You should use Arc.js whenever you want to use JavaScript or TypeScript to interact with Arc contracts. This is particularly helpful for developers who want to get the advantages of decentralized governance on the blockchain without dealing directly with a smart contract language.
+You should use Client whenever you want to use JavaScript or TypeScript to interact with Arc contracts. This is particularly helpful for developers who want to get the advantages of decentralized governance on the blockchain without dealing directly with a smart contract language.
```

Comment:
this part is related to subgraph . no ?
so : 
"You should use Subgraph service whenever you want to query our platform re  Dao states/proposals... and so forth.."